File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/main/cafemanagement/banner/BannerAction.kt
package com.hhp227.concafe.presentation.main.cafemanagement.banner

sealed interface BannerAction {
    data object ClickBack : BannerAction
    data class SelectTab(val tab: BannerTab) : BannerAction
    data object ClickCreateBanner : BannerAction
    data class ClickEditBanner(val bannerId: String) : BannerAction
    data class ClickDeleteBanner(val bannerId: String) : BannerAction
    data object ConfirmDeleteBanner : BannerAction
    data object DismissDeleteBannerDialog : BannerAction
}

File: iosApp/iosApp/Presentation/Main/CafeManagement/Banner/BannerAction.swift
//
//  BannerAction.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/15.
//

import Foundation

enum BannerAction {
    case backTapped
    case selectTab(BannerTab)
    case createBannerTapped
    case editBannerTapped(id: String)
    case deleteBannerTapped(id: String)
    case confirmDeleteBanner
    case dismissDeleteBannerDialog
}
